pyanaconda/modules/common/errors/payload.py

```diff
@@ -0,0 +1,29 @@
+#
+# Classes for exceptions used in the payload module.
+#
+# Copyright (C) 2019 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from pyanaconda.dbus.error import dbus_error
+from pyanaconda.modules.common.errors import AnacondaError
+from pyanaconda.modules.common.constants.namespaces import PAYLOAD_NAMESPACE
+
+
+@dbus_error("InvalidValuePayloadError", namespace=PAYLOAD_NAMESPACE)
+class InvalidValuePayloadError(AnacondaError):
+    """Value has incorrect supported."""
```

Comment:
Please, fix the docstring. Also, I think that we could have an exception `InvalidValueError`, that could be used by all modules. It doesn't make sense to have a generic exception only for the Payload module.

Reply:
Fixed.